src/PublicShareAuthSidebar.vue

```diff
@@ -124,6 +124,14 @@ export default {
 		})
 	},
 
+	mounted() {
+		subscribe('guest-name-added', this.postJoinConversation)
```

Comment:
Every time a guest sets their name, they will now execute postJoin

Reply:
The component is not mounted unless it is called which is with "request password" button click. It is not mounted with Talk app. 
or am I overlooking sthg ?

Reply:
yeah no that's all okay, but yeah I missed that guests don't have a rename option there.

Reply:
event can be emitted only from two components:

![image](https://github.com/nextcloud/spreed/assets/93392545/a0f8d09f-6655-4dd9-97cc-67f5024cbaec)

Technically, there is only one way (and one chance) to submit a guest name from this sidebar. Although it's better to add some guard to it, to ensure we won't trigger it again

Also guest could submit his name earlier (in one of the public chats), so please check, if that name is available